src/components/DayBoxSchedule.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from "react";
+import List, { ListItem } from "material-ui/List";
+import Typography from "material-ui/Typography";
+class DayBoxSchedule extends Component {
+  displayClasses() {
+    let card = [];
+    let divKey = 0;
+    if (
+      this.props.calendarMeeting[this.props.month] === undefined ||
+      this.props.calendarMeeting[this.props.month][this.props.day] === undefined
```

Comment:
Sorry to change it on you, but @kdjohnson and I recently decided that `Object.is` is a better way to check for this sort of thing. Also, to shorten it a bit, you can assign `this.props.calendarMeeting` to a local const that's named something shorter.